ReactAndroid/src/main/java/com/facebook/react/uimanager/ViewProps.java

```diff
@@ -157,6 +157,8 @@ public class ViewProps {
   public static final String ACCESSIBILITY_LABELLED_BY = "accessibilityLabelledBy";
   public static final String IMPORTANT_FOR_ACCESSIBILITY = "importantForAccessibility";
 
+  public static final String TEXT_CODE_BLOCK = "textCodeBlock";
```

Comment:
Maybe this could sit under `// Props that affect more than just layout`?

Reply:
It's already placed under that block, not ? I think we are good here.

Reply:
can't agree, there are two main blocks separated with comments "affect or don't affect layout". I have specifically placed it there so it's easier to distinguish in the future diffs.